src/Docker/Runner/Authorization.php

```diff
@@ -29,6 +29,12 @@ public function __construct(
     public function getAuthorization(array $configData): array
     {
         $data = [];
+
+        $appProxyAuth = $configData['app_proxy'] ?? null;
```

Comment:
Jo, to jde dobre, ze je to takhle konzervativni.

Jen mne zarazilo, ze tam neni node `authorization`, ale koukam, ze do tehle metody se posila `$configData['authorization']`, takze je to spravne.